apps/token/src/routes/governance/components/vote-details/vote-buttons.tsx

```diff
@@ -23,6 +23,7 @@ interface VoteButtonsContainerProps {
   castVote: (vote: VoteValue) => void;
```

Comment:
chore (blocking): can we add some basics tests for this please, and time box some back filling of other functionality in here